### ISSUE TITLE
Can not reproduce the effect of the paper

### ISSUE THREAD

**Issue by paulpaul91** [DevSinghSachan/ssl_text_classification#7]
Hi, Devendra:
I use your code to retrain the model, but I do not get the best result on IMDB dataset. Can you tell me how to set parameters, I just use run_aclImdbSimple_pretrained_mixed.sh.
Thank you

**Comment by DevSinghSachan** [comment]
Hi Paul, thanks for reporting. I will take a look sometime this week. Cheers,

**Comment by paulpaul91** [comment]
> Cheers

thank you

**Comment by flywangfang258** [comment]
Could you show me the link of  corresponding paper? Thank you !!!

**Comment by DevSinghSachan** [comment]
Sorry for the late reply. Here is the updated link
https://m.aaai.org/ojs/index.php/AAAI/article/view/4672

**Comment by flywangfang258** [comment]
Thank you very much！




------------------ 原始邮件 ------------------
发件人: "Devendra Singh Sachan"<notifications@github.com>;
发送时间: 2019年8月7日(星期三) 晚上11:10
收件人: "DevSinghSachan/ssl_text_classification"<ssl_text_classification@noreply.github.com>;
抄送: "王方"<592523813@qq.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [DevSinghSachan/ssl_text_classification] Can not reproduce theeffect of the paper (#7)




Sorry for the late reply. Here is the updated link
 https://m.aaai.org/ojs/index.php/AAAI/article/view/4672

**Comment by sndnyang** [comment]
Yes, I also got about "test accuracy 94.56399" with 31 epochs (slow, and I'm not sure why it stopped while it should run 50 epochs).
Any update for this?

**Comment by NealHappy** [comment]
Could you please update the result after 50 epochs? @sndnyang

**Comment by sndnyang** [comment]
> Could you please update the result after 50 epochs? @sndnyang
OK. I used the default parameters in run_aclImdbSimple_pretrained_mixed.sh , and it stopped at epoch 31. I think the reason is --nepoch_no_imprv 20.   I used  PyTorch 1.1.0

I just changed the --nepoch_no_imprv 20 to --nepoch_no_imprv 50, and run it again.  it needs about one day to finish 50 epochs. 

I run previous experiments on CUDA 10.2,  TITAN RTX.   I also had Titan XP. Now I'm running the code on Titan Xp too. I know Xp or RTX is not the issue.

**Comment by NealHappy** [comment]
> > Could you please update the result after 50 epochs? @sndnyang
> > OK. I used the default parameters in run_aclImdbSimple_pretrained_mixed.sh , and it stopped at epoch 31. I think the reason is --nepoch_no_imprv 20.   I used  PyTorch 1.1.0
> 
> I just changed the --nepoch_no_imprv 20 to --nepoch_no_imprv 50, and run it again. it needs about one day to finish 50 epochs.
> 
> I run previous experiments on CUDA 10.2, TITAN RTX. I also had Titan XP. Now I'm running the code on Titan Xp too. I know Xp or RTX is not the issue.

Thanks a lot

**Comment by sndnyang** [comment]
> Could you please update the result after 50 epochs? @sndnyang

- dev accuracy 94.59199523925781 | best dev accuracy 94.71600341796875   

No improvement.

**Comment by tbdyzm** [comment]
I run the code for ag_news datasets, I use the glove.6B.300d.txt embedding and get accuracy 92.4% after 30 epochs, which is 3% lower than the accuracy got by the paper.
